src/main/java/com/yugabyte/sample/apps/SqlConsistentHashing.java

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) YugabyteDB, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied.  See the License for the specific language governing permissions and limitations
+// under the License.
+//
+
+package com.yugabyte.sample.apps;
+
+import java.sql.Connection;
+import java.sql.Statement;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.apache.log4j.Logger;
+
+/*
+ * Consistent hashing is useful when you have a dynamic set of nodes and
+ * you need to send a key-value request to one of the nodes. Consistent
+ * hashing is great at load balancing without moving too many keys when
+ * nodes are added or removed.
+ *
+ * This app maintains a list of hashes one for each "virtual" node and
+ * supports two operations:
+ * a. Config change: Add or remove a node.
+ * b. Get node: Get the node for a given key.
+ *
+ * Config Change Operation:
+ * 1. At coin flip, choose whether to add or remove a node.
+ * 2. If adding a node, add a node with a random hash.
+ * 3. If removing a node, remove a random node.
+ *
+ * Get Node Operation:
+ * 1. Pick a random key.
+ * 2. Find the node with the smallest hash greater than the key.
+ *   If no such node exists, return the smallest hash node.
+ */
+public class SqlConsistentHashing extends AppBase {
+  private static final Logger LOG = Logger.getLogger(SqlEventCounter.class);
+
+  // Static initialization of this app's config.
+  static {
+    // Use 10 Get Node threads and 10 Config Change threads.
+    appConfig.readIOPSPercentage = -1;
+    appConfig.numReaderThreads = 10;
+    appConfig.numWriterThreads = 10;
+    // Disable number of keys.
+    appConfig.numKeysToRead = -1;
+    appConfig.numKeysToWrite = -1;
+    // Run the app for 1 minute.
+    appConfig.runTimeSeconds = 60;
```

Comment:
We are not using much of  appConfig.runTimeSeconds limitations in other workloads - do we need this in new ones?

Reply:
Other workloads insert a set number of keys and then exit. This is a run time dependent workload where the number of keys is constant and we run for a set amount for time.